doc/api/next_api_changes/removals/26910-JP.rst

```diff
@@ -0,0 +1,10 @@
+``bbox_artist``
+~~~~~~~~~~~~~~~~
+... of `matplotlib.offsetbox` is removed without replacements.
+``get_extent_offsets``
+~~~~~~~~~~~~~~~~~~~~~~~
+
+... of `matplotlib.offsetbox` is removed without replacements.
+``get_extent``
+~~~~~~~~~~~~~~~
+... of `matplotlib.offsetbox` is removed without replacements.
```

Comment:
All of these have alternatives that are mentioned in the deprecation decorator, so they are not removed "without replacements"

See the deprecation note, which can be modified to reflect the "removed" rather than "deprecated":

https://github.com/matplotlib/matplotlib/blob/602803912e7294d1882378b5601d19f5a5e7aebc/doc/api/prev_api_changes/api_changes_3.7.0/deprecations.rst?plain=1#L211-L215


https://github.com/matplotlib/matplotlib/blob/602803912e7294d1882378b5601d19f5a5e7aebc/doc/api/prev_api_changes/api_changes_3.7.0/deprecations.rst?plain=1#L191-L201

Reply:
@ksunden changed the rst file to reflect the requested changes. Please let me know if any more modifications are needed.